### PR TITLE
Revert "Revert "Add capability to filter queries #1098""

### PR DESCRIPTION
**Pull request by aeisenberg**
This reverts commit 99d4397d88749d4304bdaf50544afaf1b942c6bc.

Returns query filter functionality back into the main branch. See #1098 

### Merge / deployment checklist

- [x] Confirm this change is backwards compatible with existing workflows.
- [x] Confirm the [readme](https://github.com/github/codeql-action/blob/main/README.md) has been updated if necessary.
- [x] Confirm the [changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md) has been updated if necessary.

### CODE REVIEW

**Comment by fishfood34** [comment]
Hey there, any timeline on this? We're experiencing a noisy CodeQL rule in https://github.com/github/airflow-sources as well.

**Comment by aeisenberg** [comment]
Yes, this is on our roadmap and we will be looking at this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ No user facing changes.
 
 ## 2.1.13 - 21 Jun 2022
 
+- Add the ability to filter queries from a code scanning run by using the `query-filters` option in the code scanning configuration file. [#1098](https://github.com/github/codeql-action/pull/1098)
```
Comment (henrymercer) [inline CHANGELOG.md:32]: Might be nice to link docs and/or changelog post here once they're published.

**Review by henrymercer — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (aeisenberg) [reply CHANGELOG.md:32]: Good suggestion. When the docs are available. I will update this.

**Pull request merged**